src/components/RSVPControl.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { Link } from 'react-router'
+const { object, func } = React.PropTypes
+import { filter } from 'lodash'
+import Avatar from './Avatar'
+import Dropdown from './Dropdown'
+import { changeEventResponse } from '../actions'
+
+export default class RSVPControl extends React.Component {
+  static propTypes = {
+    currentUser: object,
+    post: object,
+    dispatch: func
+  }
+
+  render () {
+    var props = this.props
+    let { post, currentUser } = props
+
+    var yeses = filter(post.responders, er => er.response === 'yes')
```

Comment:
i'm seeing a lot of stuff here repeated 3 times -- can you just pull it out into a function and call it 3 times?